templates/redis-sentinel.conf.erb

```diff
@@ -5,7 +5,9 @@ port <%= @sentinel_port %>
 dir <%= @working_dir %>
 daemonize <%= @daemonize ? 'yes' : 'no' %>
 pidfile <%= @pid_file %>
+<% if @facts['os']['release']['major'] != '16.04' -%>
```

Comment:
This is not what I had in mind. This is much more like it:
https://github.com/voxpupuli/puppet-redis/blob/d44386133a7c524ba0f31d375cf91533f78db1cc/manifests/instance.pp#L402